Rubberduck.Core/UI/Command/ComCommands/CodeExplorerCommand.cs

```diff
@@ -18,6 +20,8 @@ public CodeExplorerCommand(CodeExplorerDockablePresenter presenter, IVbeEvents v
             _presenter = presenter;
         }
 
+        public CodeExplorerViewModel ViewModel => (_presenter.UserControl as CodeExplorerWindow)?.ViewModel;
+
```

Comment:
Why are we telling a command about a viewmodel here?

Reply:
Aye, coupling much! I'll make the vm implement some `IPeekDefinitionPopupProvider` interface, that's what it's needed for =)

Reply:
*Narrator: the vm already implements an interface named `IPeekDefinitionPopupProvider`, exactly for this. Use the interface, Luke!*

Reply:
Actually that getter has 0 references, I must have added it in order to get ahold of the VM at one point *before* the `IPeekDefinitionPopProvider` interface became a thing -- it's gone!